src/NUnitFramework/nunitlite.tests/NUnit3XmlOutputWriterTests.cs

```diff
@@ -208,6 +208,55 @@ public void TestSuite_HasValidEndTimeAttribute()
             Assert.That(success, "{0} is an invalid value for end time", endTimeString);
         }
 
+        [Test]
+        public void IgnoredTestCases_HaveValidEndTimeAttribute()
+        {
+            DateTime.TryParse(RequiredAttribute(topNode, "start-time"), out var testRunStartTime);
+            DateTime.TryParse(RequiredAttribute(topNode, "end-time"), out var testRunEndTime);
+
+            var testCaseNodes = suiteNode.SelectNodes("test-suite[@name='SkippedTest']/test-case");
+            Assert.That(testCaseNodes, Is.Not.Null);
```

Comment:
Should we also assert that Count is larger than 0 (otherwise this test could pass without actually checking the attributes).